### DISCUSSION TITLE
Concurrent Sessions showing more that it should

### DISCUSSION THREAD

**Discussion by magacho** [sipcapture/homer#440] [Q&A]
Hi everyone, I'm using Homer in an AWS ECS environment to monitor an IVR  (Asterisk-Ari and Opensips).
Everything seems to be fine, we are answering 150K calls per day, but I'm worried about some numbers:
Concurrent Sessions the numbers shown in grafana is around six time bigger than the number of Opensips and Asterisk shows. What should it be? 
Thanks.

**Comment by lmangani** [comment]
Hi @magacho 

HOMER is a raw collectors does not track sessions/transactions, just methods and allows you to do timeseries math with its metrics. If all the sources on your network are tagged with captID/etc, you should be able to tell them apart and avoid summing all the copies you might see (b2b x 2, proxy x 2, etc)
 
NOTE: If you need accurate tracking built in, HOMER's bigger brother [HEPIC](http://hepic.tel) does exactly that _(besides sponsoring 100% of HOMER)_

**Reply by magacho** [reply]
Thanks for your help I tried to contact HEPIC with no answer, can you help me with that?

**Reply by lmangani** [reply]
@magacho its the same team - just email sales@qxip.net for an appointment anytime, or let me know if needed.
